### PR TITLE
ENH: Add Thickness3D Remote module.

### PR DESCRIPTION
**Pull request by T4mmi**
This module implements filters that compute the skeleton and thickness transforms of a 3D image.

Details about the thinning/skeletonization algorithm can be found in the Insight Journal article:
Homann H.
Implementation of a 3D thinning algorithm
The Insight Journal - 2007 July - December.
http://hdl.handle.net/1926/1292
http://insight-journal.org/browse/publication/181

Medial thickness transform is a composite filter that mask the distance transform with the above thinning output.

### CODE REVIEW

File: Modules/Remote/Thickness3D.remote.cmake [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Contact: Thomas Janvier <thomas.p.janvier@gmail.com>
+itk_fetch_module(Thickness3D
+  "Tools for 3D thickness measurement"
+  GIT_REPOSITORY ${git_protocol}://github.com/InsightSoftwareConsortium/ITKBinaryThinning3D.git
```
Comment (thewtex) [inline Modules/Remote/Thickness3D.remote.cmake]: `ITKBinaryThinning3D` -> `ITKThickness3D`

File: Modules/Remote/Thickness3D.remote.cmake [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  GIT_TAG 67fe3aeb6c5c131b5d364b67565e8d895500e1e3
```
Comment (thewtex) [inline Modules/Remote/Thickness3D.remote.cmake]: Update to the latest commit, e.g. `d2bf6210f62038174aa761516635efa4436b710f`

File: Modules/Remote/Thickness3D.remote.cmake (hunk 1 above)
Comment (T4mmi) [reply Modules/Remote/Thickness3D.remote.cmake]: > `ITKBinaryThinning3D` -> `ITKThickness3D`
nice catch

**Comment by T4mmi** [comment]
I'm still not familiar with the PR/review system, I "resolved" the discution trying to integrate your changes :(
Think that the commits solved the issues...

**Comment by thewtex** [comment]
Thanks @T4mmi ! The changes look good now.

To make the `ghostflow-check-master` check pass, we need to use the standard commit subject message prefixes. You will want to check out the topic branch locally, then run `git rebase -i HEAD~3` and select `squash` for the two extra commits. For more information, see:

- https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#revise-a-topic
- https://git-scm.com/book/en/v2/Git-Branching-Rebasing

Please let us know if you have any questions! :grey_question: :racehorse:

**Comment by T4mmi** [comment]
Thanks, wow didn't know about `squash` that's awesome and would save me from erasing git history after 100x of 'try and die'...

**Comment by dzenanz** [comment]
Congratulations on getting this far! Only squashing might be needed now.

**Comment by T4mmi** [comment]
OK, I think I managed to do the squash correctly ! (*finally*)
How to "update" the PR review status ?

**Comment by dzenanz** [comment]
When you force-push, the tests are restarted. Windows tests are still running, all the others are successful.

**Comment by T4mmi** [comment]
I meant notify github that the `requested changes are addressed` 😕 
*sorry, I'm a really low level Git/Github user, still discorving its functionnalities...*

**Comment by dzenanz** [comment]
Clicking on "Resolve" is the closest thing to that.

**Comment by phcerdan** [comment]
> I meant notify github that the `requested changes are addressed`

I have been there, `Merging is blocked`, in red, is painful to see, specially after you have addressed the changes. But there is no escape. I think only the original reviewer suggesting changes can change that status! :sweat_smile:

**Comment by T4mmi** [comment]
> I have been there, `Merging is blocked`, in red, is painful to see, specially after you have addressed the changes. But there is no escape. I think only the original reviewer suggesting changes can change that status! 😅

ahah, ok ! I thought I'd missed something 😄 
no problem then, I'll leave it to @thewtex  😉

**Comment by thewtex** [comment]
@T4mmi fantastic work! Your Git Ninja powers are improving :black_large_square: .

Thank you for the contribution!

**Comment by dzenanz** [comment]
> Your Git Ninja powers are improving ⬛️ .

Is this supposed to be a black square?

**Comment by phcerdan** [comment]
I see a black square too. It is the back of the head of a ninja wearing a black hood, clearly.

**Comment by T4mmi** [comment]
seems like an actual [black square](https://emojipedia.org/black-large-square/)
>It is the back of the head of a ninja wearing a black hood
okay maybe not "just" a black square 😆

**Pull request merged**